src/Features/Traits/Console.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Features\Traits;
+
+use Illuminate\Support\Facades\Artisan;
+use PHPUnit\Framework\Assert;
+
+trait Console
+{
+    use Application;
+
+    private string $consoleOutput = "";
+
+    /**
+     * @given I run shell command :command
+     * @given I run shell command :command in console
+     */
+    public function runShellCommand(string $command): void
+    {
+        $this->consoleOutput = "";
+        exec($command, $output);
```

Comment:
I'm not sure if using exec in tests is a good idea. I would remove this method as possible violation of security of the application.